style.css

```diff
@@ -0,0 +1,634 @@
+  width: 100%;
+}
+
+.no-1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.no-1 p {
+  font-weight: bold;
+  margin: 0;
+}
+
+.lang .lang-exp .no-1 span i {
+  color: #acb7c3;
+}
+
+footer {
+  background-image: url("./img/Header-desk.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.contacts {
+  background-image: url("./img/contactshapes.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  gap: 10px;
+  width: 100%;
+  margin-top: 3rem;
+  background-repeat: no-repeat;
+  background-size: 50% 100%;
+  background-position: right;
+}
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: 0 auto 50px;
+  gap: 10px;
+  background-repeat: no-repeat;
+}
+
+.contact-intro {
+  margin-top: 80px;
+}
+
+.contact-me {
+  font-size: 2rem;
+  text-align: center;
+  color: white;
+}
+
+.paragraph {
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  color: white;
+  width: 334px;
+}
+
+.text-holder1 {
+  height: 3rem;
+  align-items: center;
+  border-radius: 0.5rem;
+  border: none;
+  padding: 12px;
+  margin-bottom: 3px;
+}
+
+.text-area {
+  color: #b3bac5;
+  height: 10rem;
+  border-radius: 0.5rem;
+  border: none;
+  padding: 12px;
+}
+
+.clicked-btn {
+  font-size: 1rem;
+  color: #396df2;
+  width: 30%;
+  padding: 15px 10px;
+  background-color: transparent;
+  border: 1px solid #396df2;
+  border-radius: 10px;
+  align-self: start;
+}
+
+@media only screen and (min-width: 768px) {
+  .navibar {
+    width: 80%;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+
+  .nav-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 10px;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  .navi-item {
+    text-align: center;
+  }
+
+  .navi-item a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .h-container {
+    display: flex;
+    gap: 10px;
+    flex-flow: column;
+    background-image: url(./img/Header-desktop.png);
+    background-size: 100%;
+  }
+
+  .h-container > * {
+    width: 70%;
+    margin: 0 auto;
+    height: auto;
+  }
+
+  .second-text {
+    order: 0;
+  }
+
+  .first-text {
+    font-size: 35px;
+  }
+
+  .hold-item {
+    width: 95%;
+    flex-flow: row;
+    justify-content: space-between;
+    gap: 20px;
+    margin: 0 auto;
+    height: 350px;
+  }
+
+  .hold-item > * {
+    height: 100%;
+  }
+
+  .hold-image {
+    width: 55%;
+  }
+
+  .hold-item section {
+    height: 100%;
+  }
+
+  .text-holder {
+    width: 100%;
+  }
+
+  .lists {
+    width: 100%;
+  }
+
+  .about-me {
+    padding: 0;
+  }
+
+  .about-me-holder {
+    display: flex;
+    flex-direction: row;
+    gap: 5%;
+    width: 100%;
+    height: auto;
+  }
+
+  .about-me-holder section {
+    flex-flow: row;
+  }
+
+  .about-me-holder section:nth-child(1) {
+    width: 45%;
```

Comment:
-  Good job achieving this 👍🏿 
- [ ] The first block which houses the  about myself div should be 40%